Configurations/unix-Makefile.tmpl

```diff
@@ -1184,20 +1184,22 @@ reconfigure reconf:
 
       if ($args{src} =~ /\.html$/) {
 	  my $title = basename($args{src}, ".html");
+	  my $pod = $args{generator}->[0];
 	  return <<"EOF";
-$args{src}: $args{generator}->[0]
+$args{src}: $pod
 	pod2html "--podroot=\$(SRCDIR)/doc" --htmldir=.. \\
-		 --podpath=man1:man3:man5:man7 "--infile=\$<" "--title=$title" \\
+		 --podpath=man1:man3:man5:man7 "--infile=$pod" "--title=$title" \\
 	| \$(PERL) -pe 's|href="http://man\\.he\\.net/(man\\d/[^"]+)(?:\\.html)?"|href="../\$1.html|g;' \\
 	> \$\@
 EOF
       } elsif ($args{src} =~ /\.(\d)$/) {
 	  my $section = $1;
 	  my $name = uc basename($args{src}, ".$section");
+	  my $pod = $args{generator}->[0];
 	  return <<"EOF";
-$args{src}: $args{generator}->[0]
+$args{src}: $pod
 	pod2man --name=$name --section=$section --center=OpenSSL \\
-		--release=\$(VERSION) \$< \\
+		--release=\$(VERSION) "$pod" \\
```

Comment:
why are quotes needed here, and not on line 1200?

Reply:
They aren't -- if ```$pod``` has spaces the dependency will be broken, and this is a relative path under our control anyway.
I guess I should have used more words in my previous comment.

Reply:
I added the quotes without even thinking about it.

However, "under our control" isn't *entirely* true.  In an out-of-source build, the relative path to the source has components that we don't control.

Reply:
then you need the quotes in the dependency line, too.

Reply:
So either add quotes or remove them, right?  That's *my* point.

Reply:
I find that overly picky (the quotes are *really* harmless), but fine, changed.